python/ray/experimental/state/common.py

```diff
@@ -516,6 +548,38 @@ class ClusterEventState(StateSchema):
     custom_fields: Optional[dict] = state_column(filterable=False, detail=True)
 
 
+class Humanify:
+    """A class containing default methods to convert units into a human readable string."""
+
+    def timestamp(x: float):
+        """Converts miliseconds to a datetime object."""
+        return datetime.datetime.fromtimestamp(x / 1000)
+
+    def memory(x: int):
+        """Converts raw bytes to megabytes"""
+        return str(x / 1e6) + " MB"
+
+    def duration(x: int):
+        """Converts miliseconds to a human readable duration."""
+        return str(datetime.timedelta(milliseconds=x))
+
+    def events(events: List[dict]):
+        """Converts a list of events into a human readable format."""
+        for event in events:
+            if "created_ms" in event:
```

Comment:
Correct me if I am wrong, I believe L1390-L1407 in common.py takes as input a TaskEvent protobuf object, but the Humanify.events helper takes as input a list of dictionaries (output of L1388), which are missing the "{...}_ts" keys.

Reply:
I see, so what are the input events here? Seems we only need to support TaskState.events right? Does it still have the `crated_ms` field? 

Or we could the `events` format function altogether if the format is too customized. I don't have a strong opinion about this,

Reply:
the input events here right now is only used for TaskState.events, it still has the created_ms field. 

My thinking was Humanify would be modified according to use case (with feedback from users), and future contributors will have a better sense of how they want to interpret the CLI output. I just made a wild guess that TaskState.events.created_ms might benefit from being humanified, and left this implementation as a future reference in case maybe another field gets added to TaskState.events and contributors can adjust the code to format it however they need.